x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/actions.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FormMessage, State } from './state';
+
+export enum ACTION {
+  ADD_REQUEST_MESSAGE = 'add_request_message',
```

Comment:
if these aren't referenced by their string values, this could probably just be a standard `enum`